packages/charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -1,9 +1,10 @@
-import * as React from 'react';
-import { classNamesFunction, find } from 'office-ui-fabric-react/lib/Utilities';
-import { IProcessedStyleSet, IPalette } from 'office-ui-fabric-react/lib/Styling';
-import { IChartProps, IHorizontalBarChartProps, IHorizontalBarChartStyles, IChartDataPoint } from './index';
-import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
 import { IHorizontalBarChartStyleProps } from '@uifabric/charting/lib/components/HorizontalBarChart/HorizontalBarChart.types';
+import { mergeStyles } from 'office-ui-fabric-react';
+import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
+import { IPalette, IProcessedStyleSet, IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { classNamesFunction, find } from 'office-ui-fabric-react/lib/Utilities';
+import * as React from 'react';
```

Comment:
maybe reorder imports so we don't show them as diff since they didn't change

Reply:
revert it to the previous version.